src/mp4atom.cpp

```diff
@@ -404,7 +404,7 @@ void MP4Atom::ReadProperties(uint32_t startIndex, uint32_t count)
             else
                oss << "atom '" << GetType() << "' is too small; overrun reading property";
 
-            throw new Exception( oss.str().c_str(), __FILE__, __LINE__, __FUNCTION__ );
+            log.verbose1f( "%s", oss.str().c_str() );
```

Comment:
Is it still safe to continue here?  ie will m_pProperties[i]->GetType() and m_pProperties[i]->Dump(...) possibly crash when called?  Maybe we should just return instead?

Reply:
I think it's okay as-is but an early return seems safer. I'll make that change.